android/src/main/java/app/edge/rnzcash/RNZcashModule.kt

```diff
@@ -140,6 +140,46 @@ class RNZcashModule(private val reactContext: ReactApplicationContext) :
                     args.putString("orchardTotalZatoshi", orchardTotalZatoshi.value.toString())
                 }
             }
+
+            wallet.status.collectWith(scope, { status ->
+                sendEvent("StatusEvent") { args ->
+                    args.putString("alias", alias)
+                    args.putString("name", status.toString())
+                }
+            })
```

Comment:
What does this do; how does it tie into the error handler? Does it add `alias` and `name` properties to the event (`SynchronizerState`)?